settings/ManualDirectDebit.setting.php

```diff
@@ -23,7 +24,7 @@
     'group_name' => 'Manual Direct Debit',
     'group' => 'manualdirectdebit',
     'name' => 'manualdirectdebit_minimum_reference_prefix_length',
-    'title' => 'Minimum mandate reference length',
+    'title' => 'Minimum Mandate Reference Length',
```

Comment:
@MiyaNoctem  since we are localizing in the settings title, can we add ts() function for all title in settings as you add them in the html attributes?

Reply:
I think this is already being done on the form (not sure if this is what you mean):

https://github.com/compucorp/uk.co.compucorp.manualdirectdebit/blob/fef4327b17b9bdb826743fe725a36155b349213e/CRM/ManualDirectDebit/Form/Configurations.php#L58